specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,560 @@
+      "type": "object",
+      "description": "This is the parameter set of either the Orchestration project itself or one of the target services.",
+      "required": [
+        "targetKind"
+      ],
+      "discriminator": "targetKind",
+      "properties": {
+        "targetKind": {
+          "type": "string",
+          "description": "The type of a target service.",
+          "enum": [
+            "luis",
+            "conversation",
+            "question_answering",
+            "non_linked"
+          ],
+          "x-ms-enum": {
+            "name": "targetKind",
+            "modelAsString": true
+          }
+        },
+        "apiVersion": {
+          "type": "string",
+          "description": "The API version to use when call a specific target service."
+        }
+      }
+    },
+    "NoneLinkedTargetIntentResult": {
+      "type": "object",
+      "description": "A wrap up of non-linked intent response.",
+      "x-ms-discriminator-value": "non_linked",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TargetIntentResult"
+        }
+      ],
+      "properties": {
+        "result": {
+          "type": "object",
+          "$ref": "#/definitions/ConversationResult",
+          "description": "The actual response from a Conversation project.",
+          "x-nullable": true
+        }
+      }
+    },
+    "LUISParameters": {
+      "description": "This is a set of request parameters for LUIS Generally Available projects.",
+      "x-ms-discriminator-value": "luis",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalysisParameters"
+        }
+      ],
+      "properties": {
+        "query": {
+          "type": "string",
+          "maxLength": 500,
+          "description": "The utterance to predict."
+        },
+        "callingOptions": {
+          "type": "object",
+          "description": "This customizes how the service calls LUIS Generally Available projects.",
+          "$ref": "#/definitions/LUISCallingOptions"
+        }
+      },
+      "additionalProperties": true
+    },
+    "LUISCallingOptions": {
+      "description": "This customizes how the service calls LUIS Generally Available projects.",
+      "type": "object",
+      "properties": {
+        "verbose": {
+          "description": "Enable verbose response.",
+          "type": "boolean"
+        },
+        "log": {
+          "description": "Save log to add in training utterances later.",
+          "type": "boolean"
+        },
+        "show-all-intents": {
+          "description": "Set true to show all intents.",
+          "type": "boolean"
+        },
+        "timezoneOffset": {
+          "type": "number",
+          "description": "The timezone offset for the location of the request."
+        },
+        "spellCheck": {
+          "type": "boolean",
+          "description": "Enable spell checking."
+        },
+        "bing-spell-check-subscription-key": {
```

Comment:
Property names should be lowerCamelCase.

Reply:
@fxs130430
Does service really use this field name?

As guideline, it should be camel case.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#json-resource-schema--field-mutability

If service does return this, you cannot fix this in the deployed api-version as it breaks. But please discuss whether to fix in in 2022-03-01-preview